bokeh/charts/builder/bar_builder.py

```diff
@@ -36,12 +36,23 @@
 
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
-        xgrid=False, ygrid=True, **kw):
+        xgrid=False, ygrid=True, continuous_range=None, **kw):
+
+    if continuous_range and not isinstance(continuous_range, Range1d):
```

Comment:
I think the validation by `Range` is enough for now... I would add more validation when the `categorical_range` is in place... but specifying a little bit can not be bad... so OK, you convinced me, LGTM...